facia/app/controllers/FaciaController.scala

```diff
@@ -180,7 +183,9 @@ trait FaciaController
           Cached(CacheTime.Facia)(JsonComponent.fromWritable(JsObject(Nil))),
         )
       } else {
-        frontJsonFapi.get(path, liteRequestType).map { resp =>
+        val requestType = if (ActiveExperiments.isParticipating(RemoveLiteFronts)) liteRequestType else fullRequestType
```

Comment:
Perhaps I'm completely misreading this but shouldn't this be the other way around? If it participates in the test we request the full front otherwise we keep on requesting the lite version.

```suggestion
        val requestType = if (ActiveExperiments.isParticipating(RemoveLiteFronts)) fullRequestType else liteRequestType
```